File: Ejercicio 25/ej25.py
phrase1 = "Hola Como estas"
phrase2 = "MUCHO MAS CON MAYuscula"

def lower_or_uper(st: str):
    m = 0
    M = 0

    for i in st:
        if i.islower():
            m += 1
        else:
            M += 1

    if m > M:
        new_p = st.lower()
    else:
        new_p = st.upper()
    return new_p

print(lower_or_uper(phrase1))    
print(lower_or_uper(phrase2))

File: Ejercicio 25/ej25.js
let phrase1 = "Hola Como estas";
let phrase2 = "MUCHO MAS CON MAYuscula";

function isUper(char){
    return char.toUpperCase() === char;
}

function isLower(char){
    return char.toLowerCase() === char;
}

function isUpperOrLower(str){
    let l1 = str.split('');
    let m = 0;
    let M = 0; 
    for (const char of l1) {
        if(isUper(char)){
            M++;
        }else if(isLower(char)){
            m++
        }
    }
    if(m > M){
        str.toLowerCase();
    }else{
        str.toUpperCase();
    }
    return str
}

console.log(isUpperOrLower(phrase1));
console.log(isUpperOrLower(phrase2));